spec/game_spec.rb

```diff
@@ -16,14 +16,91 @@
             expect(game.board).to be_a(Board)
         end
 
-        # can make game also create new instances of player, computer
+        it 'starts with a player object' do
+            game = Game.new
+
+            expect(game.player).to be_a(Player)
+        end
+
+        it 'starts with a computer object' do
+            game = Game.new
+            
+            expect(game.computer).to be_a(Computer)
+        end
+
+        it 'starts with the player as the current player' do
+            game = Game.new
+
+            expect(game.current_player).to be_a(Player)
+        end
+
+        it 'starts with 0 turns taken' do
+            game = Game.new
+
+            expect(game.turns_taken).to eq(0)
+        end
     end
 
-    describe '#welcome_message' do
-        it 'has a welcome message for the game' do
+    describe '#change_current_player & #current_player' do
+        it 'will change current player from player to computer' do
             game = Game.new
 
-            expect(game.welcome_message).to eq('Welcome to Connect Four')
+            expect(game.current_player).to be_a(Player)
+
+            game.change_current_player
+
+            expect(game.current_player).to be_a(Computer)
+        end
+
+        it 'will change current player from computer to player' do
+            game = Game.new
+
+            expect(game.current_player).to be_a(Player)
+
+            game.change_current_player
+
+            expect(game.current_player).to be_a(Computer)
+            
+            game.change_current_player
+
+            expect(game.current_player).to be_a(Player)
+        end
+    end
+
+    # The below tests for take turn pass, however the method does initiate the game to start and interrupts RSpec.
```

Comment:
I like that this test showed us that take_turn add to the count and change players and I think we can leave it commented out for documentation